HRIS.Services.Tests/Services/EmployeeSalaryDetailsServiceUnitTests.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System.Linq.Expressions;
+using HRIS.Models;
+using HRIS.Models.Enums;
+using HRIS.Services.Interfaces;
+using HRIS.Services.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Amqp.Framing;
+using MockQueryable.Moq;
+using Moq;
+using RGO.Tests.Data.Models;
+using RR.Tests.Data.Models.HRIS;
+using RR.UnitOfWork;
+using RR.UnitOfWork.Entities.HRIS;
+using Xunit;
+
+namespace HRIS.Services.Tests.Services
+{
+    public class EmployeeSalaryDetailsServiceUnitTest
+    {
+        private readonly Mock<IUnitOfWork> _unitOfWorkMock;
+        private readonly Mock<IErrorLoggingService> _errorLoggingServiceMock;
+        private readonly Mock<IEmployeeSalarayDetailsService> _employeeSalaryDetailsServiceMock;
+        private readonly EmployeeSalaryDetailsService _employeeSalaryDetailsService;
+
+        public EmployeeSalaryDetailsServiceUnitTest()
+        {
+            _unitOfWorkMock = new Mock<IUnitOfWork>();
+            _errorLoggingServiceMock = new Mock<IErrorLoggingService>();
+            _employeeSalaryDetailsServiceMock = new Mock<IEmployeeSalarayDetailsService>();
+            _employeeSalaryDetailsService = new EmployeeSalaryDetailsService(_unitOfWorkMock.Object, _errorLoggingServiceMock.Object);
+        }
+
+        int employeeId = 1;
```

Comment:
Please put these variables with the others at the top

Reply:
My bad, variables have been moved.